resources/zib/zib-Encounter.xml

```diff
@@ -223,29 +289,15 @@
         <comment value="Procedure" />
       </mapping>
     </element>
-    <element id="Encounter.reasonReference:deviatingResult">
-      <path value="Encounter.reasonReference" />
-      <sliceName value="deviatingResult" />
-      <short value="DeviatingResult" />
-      <definition value="A deviating result which serves as the reason for the contact." />
-      <comment value="DeviatingResult is captured with a reference to the zib LaboratoryTestResult instead of a string value as the zib 2020 incorrectly states. This will be fixed in a future version of the zib. At time of writing (November 2021), the proposed change can be followed in this BITS ticket: &lt;https://bits.nictiz.nl/browse/ZIB-1427&gt;." />
-      <alias value="AfwijkendeUitslag" />
-      <type>
-        <code value="Reference" />
-        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-LaboratoryTestResult" />
-      </type>
-      <mapping>
-        <identity value="zib-encounter-v4.0.1-2020EN" />
-        <map value="NL-CM:15.1.12" />
-        <comment value="DeviatingResult" />
-      </mapping>
-    </element>
     <element id="Encounter.hospitalization.admitSource">
       <path value="Encounter.hospitalization.admitSource" />
       <short value="Origin" />
       <definition value="Location from which the patient comes before the encounter. In most cases this will only be used when the patient is admitted." />
       <alias value="Herkomst" />
       <binding>
+        <extension url="http://hl7.org/fhir/StructureDefinition/elementdefinition-bindingName">
+          <valueString value="AdmitSource" />
+        </extension>

```

Comment:
Zijn dit Forge artifacts? (Zie ook verderop?)

Reply:
Jup. Heeft volgens mij te maken met de stand van de maan of dit er wel of niet in zit.